gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/BundleSet.java

```diff
@@ -35,7 +33,7 @@
     private FileSet sourceFiles;
 
     @Parameter(defaultValue = "JAVA")
```

Comment:
same comment… change constant to DefaultResourceFilterProvider.Filter.JAVA.name()?

Reply:
I'll find out. I agree that hardcoding string here is not ideal.